code/modules/projectiles/projectile/magic.dm

```diff
@@ -214,11 +214,6 @@ proc/wabbajack(mob/living/M)
 					var/datum/preferences/A = new()	//Randomize appearance for the human
 					A.copy_to(new_mob)
 
-					var/mob/living/carbon/human/human/H = new_mob
```

Comment:
Functionality loss, also, should spawn carbon/human, not carbon/human/human for consistency with the language overhaul. Then set the species as random, to keep functionality.